routes/truckRoutes.js

```diff
@@ -0,0 +1,53 @@
+var express = require('express');
+var bodyParser = require('body-parser');
+
+var Truck = require('../models/truckModel');
+
+var urlEncoded = bodyParser.urlencoded({ extended: false });
+
+var router = express.Router();
+
+router.route('/')
+  .get(function(request, response) {
+    Truck.find(function(error, results) {
+      if (error) {
+        response.status(500).send(error);
+      } else {
+        response.send(results);
+      }
+    });
+  })
+  .post(urlEncoded, function(request, response) {
+    var truck = new Truck(request.body);
+
+    truck.save(function (error, result) {
+      if (error) {
+        response.status(500).send(error);
+      } else {
+        response.status(201);
```

Comment:
You've set the correct status here, but you didn't send a response. The POST just hangs in  your handler.